main/OpenCover.Test/Framework/Persistance/BasePersistenceTests.cs

```diff
@@ -1192,5 +1192,74 @@ public void Session_Summary_Aggregates_Modules()
             Assert.AreEqual(4.12m, Instance.CoverageSession.Summary.MinCrapScore);
             Assert.AreEqual(42m, Instance.CoverageSession.Summary.MaxCrapScore);
         }
+
+        [Test]
+        public void Method_NPath_IsCalculated()
```

Comment:
This test should be two tests
1) the original test
2) and something like WhenNPathExceedsStorageCapacityDefaultsToMaxValue